.azure-pipelines/win32-conda-CI-tf-keras.yml

```diff
@@ -17,17 +18,17 @@ jobs:
       Python35:
         python.version: '3.5'
         ONNX_PATH: onnx==1.2.3
-        TENSORFLOW_PATH: tensorflow==1.11.0
+        KERAS: keras==2.1.6
 
       Python36:
         python.version: '3.6'
         ONNX_PATH: onnx==1.5.0
-        TENSORFLOW_PATH: tensorflow==1.13.1
+        KERAS: keras
```

Comment:
And it doesn't make sense to install keras for tf.keras CI tests.

Reply:
This is temporary test, will remove.